ipmininet/router/config/zebra.py

```diff
@@ -63,6 +65,7 @@ def build(self):
         cfg = super(Zebra, self).build()
         # Update with preset defaults
         cfg.update(self.options)
+        cfg.routerid = self.get_routerid()
```

Comment:
I believe router id should be handled by QuaggaDaemon directly, as it is common to every daemon (and that way will be accessible by the subclasses).